--- a/ui/base/l10n/l10n_util.h
+++ b/ui/base/l10n/l10n_util.h
@@ -58,6 +58,10 @@ std::string GetApplicationLocale(std::string_view app_locale = "");
 base::RepeatingCallback<std::string()> GetLocaleChangedCallback();
 void SetLocaleChangedCallback(
     base::RepeatingCallback<std::string()> callback);
+
+// navium: Restrict available languages to Spanish and English only
+const char kNaviumSupportedLanguages[][6] = {"en", "es"};
 
 enum TextDirection {

